arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/Iterable.kt

```diff
@@ -282,7 +283,13 @@ public inline fun <B, C, D, E, F, G, H, I, J, K, L> Iterable<B>.zip(
 internal fun <T> Iterable<T>.collectionSizeOrDefault(default: Int): Int =
   if (this is Collection<*>) this.size else default
 
-public inline fun <E, A, B> Iterable<A>.traverseEither(f: (A) -> Either<E, B>): Either<E, List<B>> {
+@Deprecated("use traverse instead", ReplaceWith("traverse(f)", "arrow.core.traverse"))
```

Comment:
```suggestion
@Deprecated("traverseEither is being renamed to traverse to simplify the Arrow API", ReplaceWith("traverse(f)", "arrow.core.traverse"))
```

Slightly more descriptive? Otherwise, people might not know what this change implies.